### PR TITLE
Fixed padding issues and other alignment issues in `Input` elements

### PR DESCRIPTION
**Pull request by vivekvijayakrishnan**
Fixed padding issues and other alignment issues for the below `Input` elements.

- Input.Text
- Input.Number
- Input.Date
- Input.Time
- Input.Toggle

### CODE REVIEW

**Review by IbrahimSulai — APPROVED** [review]
LGTM

File: source/community/reactnative/src/utils/constants.js [hunk 1]
```diff
@@ -48,6 +48,7 @@ export const WhiteColor = 'white';
 export const BlackColor = '#000000';
 export const LightBlack = "#404040";
 export const buttonDefaultColor = "#1D9BF6";
+export const BackGroudDisabledColor = "rgba(0,0,0,.3)";
```
Comment (Zaidos9) [inline source/community/reactnative/src/utils/constants.js:51]: nit: spelling

Comment (vivekvijayakrishnan) [reply source/community/reactnative/src/utils/constants.js]: thanks. changed 👍

**Review by pragadeeshk — APPROVED** [review]
LGTM

**Pull request merged**